pkg/proc/gdbserial/gdbserver_conn_arm64.go

```diff
@@ -0,0 +1,16 @@
+//+build arm64
+
+package gdbserial
+
+const (
+	regnamePC = "pc"
```

Comment:
It would probably be a good idea to define those constants somewhere else, since multiple backends might need to refer to them.